### DISCUSSION TITLE
Any reason why the build is still mentioning/looking for vulkan-renderer despite not being supported?

### DISCUSSION THREAD

**Discussion by Antiz96** [hyprwm/Hyprland#5310] [Q&A] [answered]
Hi,

When building hyprland, the build logs reports the following:

>    drm-backend     : YES
>    x11-backend     : YES
>    libinput-backend: YES
>    xwayland        : YES
>    gles2-renderer  : YES
>    **vulkan-renderer : NO**
>    gbm-allocator   : YES
>    session         : YES
>   xcb-errors      : YES
>    egl             : YES
>    libliftoff      : YES

From my understanding, it has been stated that hyprland do not support vulkan{-renderer}, e.g. in:
- https://github.com/hyprwm/Hyprland/commit/c0082519ae476cf7fb75fcb2016607d193b14c49
- https://github.com/hyprwm/Hyprland/issues/1396

If it's not supported (and never will be according to the above links), is there any reason why it is still mentioned/looked for at build time?

Thanks :)

**Comment by MightyPlaza** [comment, marked as answer]
If I'm not mistaken, that is from building wlroots, which does support vulkan

**Reply by Antiz96** [reply]
Oh right, that makes sense. Thanks
